sources/netdog/src/net_config/v3.rs

```diff
@@ -108,12 +108,13 @@ impl Validate for NetConfigV3 {
 #[cfg(test)]
 mod tests {
     use crate::net_config::test_macros::{
-        basic_tests, bonding_tests, dhcp_tests, static_address_tests, vlan_tests,
+        basic_tests, bonding_tests, dhcp_tests, mtu_tests, static_address_tests, vlan_tests,
     };
 
     basic_tests!(3);
     dhcp_tests!(3);
     static_address_tests!(3);
     vlan_tests!(3);
     bonding_tests!(3);
+    mtu_tests!(3);
```

Comment:
We'll need to increment the schema version for the MTU-related changes:
```suggestion
    mtu_tests!(4);
```